model/comment.py

```diff
@@ -9,5 +9,11 @@ class Comment(db.Model):
     comment = db.Column(db.String, nullable=False)
     timestamp = db.Column(db.DateTime, nullable=False, default=datetime.datetime.utcnow)
 
-    object_id = db.Column(db.Integer, db.ForeignKey('object.id'))
-    user_id = db.Column(db.Integer, db.ForeignKey('user.id'))
+    object_id = db.Column(db.Integer, db.ForeignKey('object.id'), nullable=False)
+    user_id = db.Column(db.Integer, db.ForeignKey('user.id'), nullable=False)
```

Comment:
This adds additional constraints without sql migrations. Are these columns already `NOT NULL`?

Reply:
Completely forgot about that. Migration added.